dbt_loom/__init__.py

```diff
@@ -150,7 +150,7 @@ def convert_model_nodes_to_model_node_args(
         unique_id: ModelNodeArgs(
             name=node.get("name"),
             package_name=node.get("package_name"),
-            identifier=node.get("relation_name").split(".")[-1].replace('"', ""),
+            identifier=node.get("relation_name").split(".")[-1].replace('"', "").replace('`',""),
```

Comment:
Fixed a bug where the identifier would have double backticks.

![Screenshot 2023-11-08 at 10 08 49](https://github.com/nicholasyager/dbt-loom/assets/148971723/94fe983e-2b68-4a0b-a169-b8ae0de868de)